barcodes_search/wizards/barcode_search_line.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import ast
+from odoo import _, api, fields, models
+
+
+class BarcodeSearchLine(models.TransientModel):
+    _name = 'barcode.search.line'
+
+    barcode_search_id = fields.Many2one(
+        comodel_name='barcode.search')
+
+    model_id = fields.Many2one(
+        comodel_name='ir.model', readonly=True, related='field_id.model_id')
+
+    field_id = fields.Many2one(
+        comodel_name='ir.model.fields', readonly=True, string='Field')
+
+    item_id = fields.Integer(string='Item ID', readonly=True)
```

Comment:
Can we name this differently? I had to jump back up because I thought it was an actual ID column. Maybe `item_identifier`? or `item_ref`?